tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -22,8 +22,10 @@ namespace tensorflow {
 
 namespace {
 
-// In case of failure, every call will be retried kMaxAttempts-1 times.
-constexpr int kMaxAttempts = 4;
+// In case of failure, every call will be retried kMaxAttempts times.
+constexpr int kMaxAttempts = 3;
```

Comment:
Actually then 'attempt' should become 'retries' (not 'retry') and will represent the amount of retries made so far, so it will make sense that it is  0 in the beginning.